### PR TITLE
Documentation Updates for standalone build

### PR DESCRIPTION
**Pull request by karthikamurthy**
Updated BUILD.md and PREREQUISITES.md for setting up standalone build environment.

Updated tcs_standup.sh and Demo.py for running test on standalone avalon build.

Signed-off-by: Karthika Murthy <karthika.murthy@intel.com>

### CODE REVIEW

File: tests/Demo.py [hunk 1]
```diff
@@ -100,7 +100,8 @@ def local_main(config):
                     # response and update the worker id information for
                     # further json requests.
                     if "result" in response and \
-                            "ids" in response["result"].keys():
+                            "ids" in response["result"].keys() and \
+                            len(response["result"]["ids"]) > 0:
```
Comment (danintel) [inline tests/Demo.py:104]: This should be in a separate commit.

File: scripts/tcs_startup.sh [hunk 2]
```diff
@@ -86,7 +86,16 @@ stop_avalon_components()
     exit
 }
 
-while getopts "l:styh" OPTCHAR ; do
+stop_avalon_components_forcefully()
```
Comment (danintel) [inline scripts/tcs_startup.sh:89]: This should be in a separate commit

File: PREREQUISITES.md [hunk 3]
```diff
@@ -397,7 +399,7 @@ problems.
 # <a name="troubleshooting"></a>Troubleshooting Installation
 - Verify your [environment variables](#environment) are set correctly and the
   paths exist
-
+- Developers are suggested for to use openssl1.1.1d version due to some technical issues.
```
Comment (danintel) [inline PREREQUISITES.md:402]: I would reword this to:

- Verify you are using OpenSSL 1.1.1d with `openssl version`

File: BUILD.md [hunk 4]
```diff
@@ -136,14 +136,23 @@ The steps below will set up a Python virtual environment to run Avalon.
    If you are not behind a corporate proxy (the usual case),
    then leave this line commented out.
 
-7. Create a Python virtual environment:
+7. Standalone build does not support communication between listener and enclave manager over the ZMQ.
+   In file `$TCF_HOME/config/singleton_enclave_config.toml set sync_workload_execution flag to 0
```
Comment (Ram-srini) [inline BUILD.md:140]: We need to fix listener issue by changing the zmq url to localhost that will fix actual issue with ZMQ.

File: BUILD.md [hunk 5]
```diff
@@ -153,13 +162,13 @@ The steps below will set up a Python virtual environment to run Avalon.
    If the virtual environment for the current shell session is activated,
    you will the see this prompt: `(_dev)`
 
-9. Install PIP3 packages into your Python virtual environment:
+10. Install PIP3 packages into your Python virtual environment:
 
    ```bash
-   pip3 install --upgrade setuptools json-rpc py-solc-x web3 colorlog twisted wheel toml
+   pip3 install --upgrade setuptools json-rpc py-solc-x web3 colorlog twisted wheel toml pyzmq pycrypto
```
[the remark this answers is not in the document]
Comment (danintel) [reply BUILD.md:168]: A separate PR will add pycrypto and the change should be made in that PR, not here.

File: scripts/tcs_startup.sh (hunk 2 above)
Comment (karthikamurthy) [reply scripts/tcs_startup.sh:89]: Committed this change separately. Done

File: tests/Demo.py (hunk 1 above)
Comment (karthikamurthy) [reply tests/Demo.py:104]: Done

File: PREREQUISITES.md [hunk 6]
```diff
@@ -45,7 +45,7 @@ activation script (e.g. `source /opt/intel/sgxsdk/environment`)
 - If you build your own OpenSSL (not the usual case),
   `PKG_CONFIG_PATH` and `LD_LIBRARY_PATH` also contain the the path to
   [OpenSSL](#openssl) package config files and libraries, respectively.
-  You need to do this when pre-built OpenSSL version 1.1.1d or later
+  You need to do this when pre-built OpenSSL version 1.1.1f or later
```
Comment (pankajgoyal2) [inline PREREQUISITES.md:48]: Why are you changing openssl version version from 1.1.1d to 1.1.1f?

Comment (karthikamurthy) [reply PREREQUISITES.md:48]: When compiling Python module in standalone build is reporting that the ssl module failed to compile due to missing flags despite it existing for version OpenSSL 1.1.1d. No such error occurred in Openssl1.1.1f for standalone build.
